src/detection/card/items/date-format.js

```diff
@@ -0,0 +1,6 @@
+export const DateFormat = timestamp => {
```

Comment:
Does this function *really* need to be a component? :smile:

Reply:
done